content/docs/user-guide/integrations/evidently.md

```diff
@@ -0,0 +1,192 @@
+
+df = pd.read_csv("raw_data/day.csv", header=0, sep=',', parse_dates=['dteday'])
+df.head()
+```
+
+This is how it looks: ![Sample data](/img/evidently-sample-data-head.png)
+
+## Define column mapping
+
+You should specify the categorical and numerical features so that Evidently
+performs the correct statistical test for each of them. While Evidently can
+parse the data structure automatically, manually specifying the column type can
+minimize errors.
+
+```python
+from evidently.pipeline.column_mapping import ColumnMapping
+
+data_columns = ColumnMapping()
+data_columns.numerical_features = ['weathersit', 'temp', 'atemp', 'hum', 'windspeed']
+data_columns.categorical_features = ['holiday', 'workingday']
+```
+
+## Define what to log
+
+Specify which metrics you want to calculate. In this case, you can generate the
+Data Drift report and log the drift score for each feature.
+
+```python
+from evidently.report import Report
+from evidently.metric_preset import DataDriftPreset
+
+
+def eval_drift(reference, production, column_mapping):
+    data_drift_report = Report(metrics=[DataDriftPreset()])
+    data_drift_report.run(
+        reference_data=reference, current_data=production, column_mapping=column_mapping
+    )
+    report = data_drift_report.as_dict()
+
+    drifts = []
+
+    for feature in (
+        column_mapping.numerical_features + column_mapping.categorical_features
+    ):
+        drifts.append(
+            (
+                feature,
+                report["metrics"][1]["result"]["drift_by_columns"][feature][
+                    "drift_score"
+                ],
+            )
+        )
+
+    return drifts
+
+```
+
+You can adapt what you want to calculate by selecting a different Preset or
+Metric from those available in Evidently.
+
+## Define the comparison windows
+
+Specify the period that is considered reference: Evidently will use it as the
+base for the comparison. Then, you should choose the periods to treat as
+experiments. This emulates the production model runs.
+
+```python
+#set reference dates
+reference_dates = ('2011-01-01 00:00:00','2011-01-28 23:00:00')
+
+#set experiment batches dates
+experiment_batches = [
+    ('2011-01-01 00:00:00','2011-01-29 23:00:00'),
+    ('2011-01-29 00:00:00','2011-02-07 23:00:00'),
+    ('2011-02-07 00:00:00','2011-02-14 23:00:00'),
+    ('2011-02-15 00:00:00','2011-02-21 23:00:00'),
+]
+```
+
+## Run and log experiments in DVC
+
+There are two ways to track the results of Evidently with DVCLive:
+
+1. you can save the results of each item in the batch in one single experiment
+   (each experiment corresponds to a git commit), in separate steps
+2. or you can save the result of each item in the batch as a separate experiment
+
+We will demonstrate both, and show you how to inspect the results regardless of
+your IDE. However, if you are using VSCode, we recommend using
+[our awesome VSCode extension](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
```

Comment:
```suggestion
the [DVC extension for VS Code](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
```

Appreciate the love but technically this is what we are supposed to call it!

Reply:
done